tools/cooler/cooler_zoomify.xml

```diff
@@ -0,0 +1,41 @@
+<tool id="cooler_zoomify" name="Cooler zoomify" version="@VERSION@+galaxy0" profile="18.01">
+  <description>Generate a multi-resolution cooler file by coarsening</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements" />
+  <expand macro="stdio" />
+  <command detect_errors="exit_code"><![CDATA[
+    cooler zoomify -p \${GALAXY_SLOTS:-4}
+    -c $c
+    -r $r
```

Comment:
```suggestion
    -r '$r'
```

Reply:
or is it because it's a string?

Reply:
All "path" or "strings"/text needs to be single quoted. Imagine there is a space included, e.g. the admin has deployed Galaxy on a path with a space included or your string has a space. Or your text can be inflated via an ENV var. All those cases are potential problems, security issues etc. Thats why we recommend to single-quote them - not double quote, double quotes are evaluated by bash, e.g. $ENV expansion will happen.